core/Cargo.toml

```diff
@@ -235,6 +236,7 @@ suppaftp = { version = "4.5", default-features = false, features = [
   "async-secure",
   "async-rustls",
 ], optional = true }
+tikv-client = { version = "0.2.0", optional = true }
```

Comment:
You can use git version to develop first. Change to tagged version before merging.